app/src/main/res/values/strings.xml

```diff
@@ -79,174 +79,174 @@
-    <string name="reader_mode_hint">Chosen configuration will be remembered for this manga</string>
+    <string name="reader_mode_hint">The chosen configuration will be remembered for this manga</string>
     <string name="silent">Silent</string>
-    <string name="captcha_required">CAPTCHA is required</string>
+    <string name="captcha_required">CAPTCHA required</string>
     <string name="captcha_solve">Solve</string>
     <string name="clear_cookies">Clear cookies</string>
-    <string name="cookies_cleared">All cookies was removed</string>
+    <string name="cookies_cleared">All cookies were removed</string>
     <string name="chapters_checking_progress">Checking for new chapters: %1$d of %2$d</string>
     <string name="clear_feed">Clear feed</string>
-    <string name="text_clear_updates_feed_prompt">All updates history will be cleared and this action cannot be undone. Are you sure?</string>
-    <string name="new_chapters_checking">New chapters checking</string>
+    <string name="text_clear_updates_feed_prompt">Clear all update history permanently?</string>
+    <string name="new_chapters_checking">Looking for new chapters…</string>
     <string name="reverse">Reverse</string>
     <string name="sign_in">Sign in</string>
-    <string name="auth_required">You should authorize to view this content</string>
+    <string name="auth_required">Sign in to view this content</string>
     <string name="default_s">Default: %s</string>
     <string name="_and_x_more">…and %1$d more</string>
     <string name="next">Next</string>
-    <string name="protect_application_subtitle">Enter password that will be required when the application starts</string>
+    <string name="protect_application_subtitle">Enter a password to start the app with</string>
     <string name="confirm">Confirm</string>
-    <string name="password_length_hint">Password must be at least 4 characters</string>
+    <string name="password_length_hint">The password must be 4 characters or more</string>
     <string name="hide_toolbar">Hide toolbar when scrolling</string>
     <string name="search_only_on_s">Search only on %s</string>
-    <string name="text_clear_search_history_prompt">Do you really want to remove all recent search queries? This action cannot be undone.</string>
+    <string name="text_clear_search_history_prompt">Remove all recent search queries permanently?</string>
     <string name="other">Other</string>
     <string name="languages">Languages</string>
     <string name="welcome">Welcome</string>
     <string name="description">Description</string>
-    <string name="backup_saved">Backup saved successfully</string>
-    <string name="tracker_warning">Some manufacturers can change the system behavior, which may breaks background tasks.</string>
+    <string name="backup_saved">Backup saved</string>
+    <string name="tracker_warning">Some devices have different system behavior, which may break background tasks.</string>
```

Comment:
I think original meaning is lost

Reply:
Could this be explained?
I was trying to not invoke "manufacturer", since I am guessing the real issue is the Android version, or some setup done by OEMs/vendors.

Reply:
This is about OEM`s modifications